src/components/modals/ExtractConflictModal.tsx

```diff
@@ -42,7 +43,7 @@ const ExtractConflictModal: React.SFC<Props> = (props) => {
                 <p>{formatMessageId(intl, FM.EXTRACTCONFLICTMODAL_SUBTITLE)}</p>
                 <div>{formatMessageId(intl, FM.EXTRACTCONFLICTMODAL_REVIEW)}</div>
 
-                <div className="cl-inconsistent-entity-modal-header cl-text--error"><OF.Icon iconName="ChromeClose" />&nbsp;{formatMessageId(intl, FM.EXTRACTCONFLICTMODAL_CONFLICTING_LABELS)}</div>
+                <div className="cl-inconsistent-entity-modal-header cl-text--error" data-testid="entity-labels-conflict-submitted"><OF.Icon iconName="ChromeClose" />&nbsp;{formatMessageId(intl, FM.EXTRACTCONFLICTMODAL_CONFLICTING_LABELS)}</div>
```

Comment:
These wording seems misleading / backwards to me.  Prefer to keep the ids to logically match the graphic;
![image](https://user-images.githubusercontent.com/2856501/54224458-c87eca80-44b6-11e9-8009-e6522c91f4df.png)

E.g.  Conflicting vs Previously Submitted

The top input which was rejected due to the conflicts, the bottom was previously submitted and accepted.  Technically they are both submitted but one only succeeded so submitted word doesn't differentiate

Reply:
I'm creating new test automation code against this new functionality, and need a direct way to access these UI elements in keeping with our existing test code (vs walking the DOM).

Reply:
Right... we use `data-testid` from the tests. I know why you're doing it but disagree with how/when you're doing it.